.github/workflows/trigger-e2e-tests.yml

```diff
@@ -28,19 +32,51 @@ jobs:
       - name: Clean up disk space
         uses: ./.github/cleanup-disk-space
 
-      - name: Run tests
+      - name: Run E2E tests
         run: |
-          ./run-env-and-tests.sh --env ${{ matrix.env }} --elk ${{ matrix.version }}
+          ./main.sh --mode e2e --env ${{ matrix.env }} --elk ${{ matrix.version }}
         env:
           ROR_ACTIVATION_KEY: ${{ secrets.ROR_KBN_LICENSE }}
           ELECTRON_EXTRA_LAUNCH_ARGS: '--disable-gpu'
+
       - name: S3 Upload Videos & show logs
         if: failure()
         uses: ./.github/upload-videos
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # ==========================================
+  # BOOTSTRAP TESTS
+  # ==========================================
+  master-bootstrap-tests:
+    name: "🚀 Bootstrap Tests"
+    if: github.event_name == 'schedule' || github.ref == 'refs/heads/master' || github.base_ref == 'master'
+    needs: master-e2e-tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["9.1.1", "9.0.5", "8.19.2", "8.18.5", "8.17.10", "8.16.6", "8.15.5", "8.14.3", "8.13.4", "8.12.2", "8.11.4", "8.10.4", "8.9.2", "8.8.2", "8.7.1", "8.6.2", "8.5.3", "8.4.3", "8.3.3", "8.2.3", "8.1.3", "8.0.1", "7.17.29", "7.16.3", "7.15.2", "7.14.2", "7.13.4", "7.12.1", "7.11.2", "7.10.2", "7.9.3"]
```

Comment:
In like 99% cases, Kibana crashes during initialization, because of a node compatibility problem with features used by dependencies (upgraded or added).

As I see, there can be a difference between required node engines between patch versions of specific minor version, for example:

Kibana 7..9.0: Node engine 10.21.0
Kibana 7.9.3: Node engine 10.22.1

Kibana 7.17.0: Node engine 16.13.2
Kibana 7.17.29: Node engine 20.19.2

WDYT about declaring the lowest patch version of each minor version to be able to verify the lowest possible node engine for specific minor version?